examples/FSharp/FSharp.fsproj

```diff
@@ -58,4 +58,8 @@
       <PackageReference Include="xunit.runner.visualstudio" Version="2.4.3" />
     </ItemGroup>
 
+    <ItemGroup>
```

Comment:
Is it really needed?

Reply:
Good question. Actually not, but that's a chicken and egg problem.
Until changes are not in the NuGet, example project takes last NBomber NuGet and can't compile.
`ProjectReference` instead of `PackageReference` would solve this and allow to test new changes on CI before they are published. 
But maybe there is another reason why it is like it is.